libs/sr/theme/src/Theme.tsx

```diff
@@ -0,0 +1,259 @@
+import { createTheme } from '@mui/material'
+
+import { theme as commonTheme } from '@genshin-optimizer/common/ui'
+
+declare module '@mui/material/styles' {
+  interface Palette {
+    brandSRO500: Palette['primary']
+    neutral100: Palette['primary']
+    neutral200: Palette['primary']
+    neutral300: Palette['primary']
+    neutral400: Palette['primary']
+    neutral500: Palette['primary']
+    neutral600: Palette['primary']
+    neutral700: Palette['primary']
+    neutral800: Palette['primary']
+    neutral900: Palette['primary']
+
+    roll1: Palette['primary']
```

Comment:
roll colors and rarity colors may not make sense in SRO, I would recommend removing them from SRO theme for now, and we can add them later once we need them.

Reply:
I guess technically roll colours do make sense here, since in SRO there is 1 base roll, and 5 additional rolls, it would make sense to have up to roll6. I would then advocate to have it be separated from GO, and be kept in SRO for now.